github_rules/github_repo_hook_modified.py

```diff
@@ -0,0 +1,11 @@
+def rule(event):
+    return event.get("action").startswith("hook.")
+
+
+def title(event):
+    action = "modified"
+    if event.get("action").endswith("destroy"):
+        action = "deleted"
+    elif event.get("action").endswith("create"):
```

Comment:
I wonder if creating webhooks should have a higher severity (perhaps `MEDIUM` as that may indicate that someone is creating programmatic access, which could be cause for concern if they're trying to e.g. persist temporary credentials they've gotten.